assets/js/components/Layout/Layout.jsx

```diff
@@ -89,23 +89,24 @@ const Layout = () => {
                 onClick={handleSidebar}
                 title={isCollapsed ? 'Expand Sidebar' : 'Collapse Sidebar'}
               >
-                <EosKeyboardDoubleArrowLeft
-                  className="text-gray-400 hover:text-gray-300"
-                  flip={isCollapsed}
+                <EOS_KEYBOARD_DOUBLE_ARROW_LEFT
+                  size="24"
+                  color="currentColor"
+                  className={classNames('text-gray-400 hover:text-gray-300', {
+                    '-scale-100': isCollapsed,
```

Comment:
Is this `scale` thing to change arrows direction?
Other option I guess would be to use `EOS_KEYBOARD_DOUBLE_ARROW_RIGHT` depending on the `isCollapsed` value, instead of this direction change. Both work